earthmover/nodes/node.py

```diff
@@ -69,6 +70,11 @@ def __init__(self, name: str, config: 'YamlMapping', *, earthmover: 'Earthmover'
         # Always check for debug and expectations
         self.debug = self.debug or self.config.get('debug', False)
         self.expectations = self.error_handler.assert_get_key(self.config, 'expect', dtype=list, required=False)
+        self.require_rows = int(self.require_rows or self.config.get('require_rows', False))
+        if self.require_rows < 0:
+            self.error_handler.throw(
+                f"Source `{self.full_name}` require_rows cannot be negative"
```

Comment:
Will the operation information be injected automatically from the error handler?

Reply:
Yes.
> Exception: (near line 395 of `/.../repos/earthmover/earthmover/tests/earthmover.yaml` in `$transformations.families.operations:keep_columns`)  Source `$transformations.families` failed require_rows >= 1000000` (only 4 rows found))